platform/mds/mds/src/main/java/org/motechproject/mds/annotations/Discriminated.java

```diff
@@ -0,0 +1,13 @@
+package org.motechproject.mds.annotations;
+
+import javax.jdo.annotations.IdentityType;
+import javax.jdo.annotations.PersistenceCapable;
+import java.lang.annotation.*;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@PersistenceCapable(identityType = IdentityType.DATASTORE, detachable = "true")
+public @interface Discriminated {
```

Comment:
This needs javadoc first of all. Moreover I am wondering about the purpose of this annotation. Why not use the JDO inheritance annotations? Is the UI behavior the main difference? Am I understanding this change correctly?